coralproject/talk
# [CORL-2687] Go To Conversation aria label

## What does this PR do?
This PR adds an aria label to the "Go to conversation" button in a featured comment.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. As a moderator, feature a comment
2. Inspect the button in the lower right hand corner of the featured comment that says "Go to conversation"
3. Observe that it has an aria label in the form of "Go to this featured comment by user { $username } in the main comment stream" (or "Go to this featured comment in the main comment stream" in the case that the user has no username)
 
## How do we deploy this PR?
No special considerations should be required.
